metpy/calc/thermo.py

```diff
@@ -52,6 +52,37 @@ def relative_humidity_from_dewpoint(temperature, dewpt):
     return (e / e_s)
 
 
+@exporter.export
+@check_units('[pressure]')
+def exner_function(pressure):
```

Comment:
I think I'd argue for the reference pressure to default to P0, but be a kwarg incase someone has reason to change it. Thoughts @dopplershift? Maybe `reference_pressure`

Reply:
I think that's fair.